workflows/raredisease.nf

```diff
@@ -234,8 +234,8 @@ workflow RAREDISEASE {
                                                                            : Channel.empty()
     ch_intervals_y              = params.intervals_y                       ? Channel.fromPath(params.intervals_y).collect()
                                                                            : Channel.empty()
-    ch_ml_model                 = params.variant_caller.equals("sentieon") ? Channel.fromPath(params.ml_model).collect()
-                                                                           : Channel.value([])
+    ch_ml_model                 = params.variant_caller.equals("sentieon") ? Channel.fromPath(params.ml_model).map {it -> [[id:it[0].simpleName], it]}.collect()
```

Comment:
Just curious, when using `params.variant_caller.equals("sentieon")` it kind of indicates that the raredisease-pipeline is designed to only use one variant-caller per run of the pipeline - is that correct?

Reply:
Yes, that is correct 😄